### PR TITLE
[MU3] Don't show empty Inspector

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
for tremolos between notes if on tablature or not on half notes.
Also disable Inspector for system- and page breaks.

Resolves: issue discussed in the developers' chat on Telegram.

### CODE REVIEW

**Comment by Howard-C** [comment]
Just do a `for` loop for all tremolos

**Comment by Jojo-Schmitz** [comment]
> Just do a `for` loop for all tremolos

?

**Comment by Jojo-Schmitz** [comment]
That's exactly the complication we agreed on to not be worth the effort, as far as understood, isn't is?

File: mscore/inspector/inspector.cpp [hunk 1]
```diff
@@ -320,14 +320,17 @@ void Inspector::update(Score* s)
                         case ElementType::LAYOUT_BREAK:
                               if (toLayoutBreak(element())->layoutBreakType() == LayoutBreak::Type::SECTION)
                                     ie = new InspectorSectionBreak(this);
-                              else
-                                    ie = new InspectorBreak(this);
+                              //else
+                              //      ie = new InspectorBreak(this);  // currently enpty and such not needed
```
Comment (Asmatzaile) [inline mscore/inspector/inspector.cpp:324]: typo?

Comment (Jojo-Schmitz) [reply mscore/inspector/inspector.cpp:324]: yes

Comment (Jojo-Schmitz) [reply mscore/inspector/inspector.cpp]: fixed

**Comment by Howard-C** [comment]
> That's exactly the added complexity we agreed on to not be worth the effort, as far as understood, isn't is?
> 
> Getting it to work on a list selection containing 'proper' and 'improper' elements would be relatively complex

Without the complex check it's still possible to get an empty section sometimes, unless you delete the same kind of check for the specific option in `InspectorTremolo::setElement()`

**Comment by Jojo-Schmitz** [comment]
> it's still possible to get an empty section sometimes

How?

**Comment by Howard-C** [comment]
If the first element is applicable but the second is not, the section would be shown, but due to the check in `InspectorTremolo::setElement()`, the style option is invisible, which is the only option

**Comment by Jojo-Schmitz** [comment]
I see...

**Pull request merged**